examples/CMakeLists.txt

```diff
@@ -29,9 +29,11 @@
 # ==============================================================================
 # sleepy-demo
 # ==============================================================================
-option(EFR32_APP_SLEEPY_DEMO "enable sleepy-demo app" ON)
+option(EFR32_APP_SLEEPY_DEMO_FTD "enable sleepy-demo-ftd app" OFF)
```

Comment:
Because we have different platform libraries for each sample app, we need to run a separate CMake build for each platform library, each in its own build folder. Having these options default to `OFF` allows us to turn on only the target definition for the app we care about. It leaves the build folders tidier and prevents users from building a sleepy-demo app with the wrong `openthread-efr32-soc-*` platform library